src/core/public/chrome/ui/header/elastic_mark.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { HTMLAttributes } from 'react';
+
+export const ElasticMark = ({ ...props }: HTMLAttributes<SVGElement>) => (
+  <svg
+    width="64"
+    height="19"
+    viewBox="0 0 64 19"
+    fill="currentColor"
+    xmlns="http://www.w3.org/2000/svg"
+    aria-labelledby="elasticMark"
+    {...props}
+  >
+    <title id="elasticMark">Elastic</title>
+    <path d="M9.73938 16.8818L10.4515 16.8061L10.497 18.2606C8.61817 18.5182 7.01211 18.6545 5.67878 18.6545C3.90605 18.6545 2.64847 18.1394 1.90605 17.1091C1.16362 16.0788 0.799988 14.4727 0.799988 12.3061C0.799988 7.97272 2.52726 5.80606 5.96665 5.80606C7.63332 5.80606 8.87575 6.27575 9.69393 7.2C10.5121 8.12424 10.9212 9.59394 10.9212 11.5788L10.8151 12.9879H2.66362C2.66362 14.3515 2.90605 15.3667 3.40605 16.0182C3.90605 16.6697 4.75453 17.003 5.98181 17.003C7.22423 17.0333 8.46666 16.9879 9.73938 16.8818ZM9.07271 11.5333C9.07271 10.0182 8.83029 8.94242 8.34544 8.32121C7.8606 7.7 7.07272 7.38182 5.98181 7.38182C4.8909 7.38182 4.05756 7.71515 3.51211 8.36666C2.96665 9.01818 2.67878 10.0788 2.66362 11.5333H9.07271Z" />
```

Comment:
Nit: would you mind to reduce SVG size with [svgo](https://jakearchibald.github.io/svgomg/) or another tool?

Reply:
This is done.